### PR TITLE
Add progress bar

### LINKED ISSUE #24: Add progress bar
There's a really nice and subtle progress bar in the design. We should add this for student sanity. The progress should indicate how many chapters the student has finished, not where they are on the page.

screen shot 2017-01-06 at 8 28 38 am

Feel free to check out the full design here: 
 https://www.dropbox.com/s/yh8zqmaj2xlwq7o/2016-12-06.pdf?dl=0

### PR DESCRIPTION
**Pull request by alexfu**
Fixes #24

### CODE REVIEW

File: _chapters/chapter_01.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ url: 1.html
 number: 1
 title: Getting Started from Scratch
 layout: rails_tutorial
+chapter: 1
```
Comment (latazzajones) [inline _chapters/chapter_01.md]: I think this `chapter` data attribute might be redundant to the `number` data attribute.

**Review by latazzajones — CHANGES_REQUESTED** [review]
This looks great! I left one small comment - otherwise, I look forward to merging it :-)

File: _chapters/chapter_01.md (hunk 1 above)
Comment (alexfu) [reply _chapters/chapter_01.md]: Oh wow, I didn't realize that. Thanks, i'll make that change.

Comment (alexfu) [reply _chapters/chapter_01.md]: @latazzajones updated

**Pull request merged**